DiegoFlores/if-else.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class Solution {
+
+
+
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        int number = scanner.nextInt();
+        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+        if(number%2 == 1 || ( number > 6 && number <= 20 )){
+            System.out.println("Weird");
+        }else if( number > 2 && number < 5  || (number > 20)){
+            System.out.println("Not Weird");
```

Comment:
Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings